src/Microsoft.VisualStudio.ProjectSystem.Managed/Threading/Tasks/ITaskDelayScheduler.cs

```diff
@@ -28,5 +28,11 @@ internal interface ITaskDelayScheduler : IDisposable
         /// </remarks>
         /// <exception cref="ObjectDisposedException">The object has been disposed.</exception>
         JoinableTask ScheduleAsyncTask(Func<CancellationToken, Task> operation, CancellationToken token = default);
+
+        /// <summary>
+        /// Runs an asynchronous operation immediately, cancelling any previously scheduled tasks.
+        /// See <see cref="ScheduleAsyncTask(Func{CancellationToken, Task}, CancellationToken)"/> for more information.
+        /// </summary>
+        JoinableTask RunAsyncTask(Func<CancellationToken, Task> operation, CancellationToken token = default);
```

Comment:
`RunAsyncTask` can also throw `ObjectDisposedException`. Consider copying the `<exception>` documentation element from `ScheduleAsyncTask` above.